base/special/hyperbolic.jl

```diff
@@ -0,0 +1,174 @@
+# ====================================================
+# Copyright (C) 1993 by Sun Microsystems, Inc. All rights reserved.
+#
+# Developed at SunSoft, a Sun Microsystems, Inc. business.
+# Permission to use, copy, modify, and distribute this
+# software is freely granted, provided that this notice
+# is preserved.
+# ====================================================
+
+_ldexp_exp(x::Float64, i::T) where T = ccall(("__ldexp_exp", libm), Float64, (Float64, T), x, i)
```

Comment:
Just a quick comment, this should just use the julia ldexp function, if you don't need the error handling, you can  write a `_ldexp` function and remove the error/exception  handling in `ldexp`

Reply:
Oh! I looked for a Julia binding/version.. but I must have mistyped the name when searching.

Reply:
Is this worth changing now?

Reply:
Unfortunately, this function is actually *not* in Julia yet, but yes, it is worth changing, and it should be a simple function to port.

Reply:
is [this](https://github.com/JuliaLang/julia/blob/76e5232c104f8dedb329948b1d04ae8a3757143f/base/math.jl#L537) not the function @musm was suggesting?

Reply:
no, that computes `x*2^n`, what is needed here is `exp(n)*2^n`